packages/sanity/src/core/releases/navbar/GlobalPerspectiveMenuItem.tsx

```diff
@@ -170,7 +170,7 @@ export const GlobalPerspectiveMenuItem = forwardRef<
             </Text>
             {!isPublishedPerspective(release) &&
               release.metadata.releaseType !== 'undecided' &&
-              (release.publishAt || release.metadata.intendedPublishAt) && (
```

Comment:
These aren't a like for like swap I don't think, since `isScheduledRelease` only looks at `intendedPublishAt`. I can't recall exactly but it might be worth verifying that when a scheduled release is actually scheduled, and it's possible to reconfirm the scheduled datetime, whether we also update the `intendedPublishAt` to reflex this too.
<img width="385" alt="Screenshot 2024-11-05 at 13 40 15" src="https://github.com/user-attachments/assets/7613e7e1-3252-4fdd-bbbe-65135f498182">

Reply:
Are we removing the `intendedPublishAt` when scheduling it? 
It's confusing how close this names are. 

Maybe we need to find a better name for the `releaseType ="scheduled"`

Reply:
I think we leave `intendedPublishAt` as whatever it was, and only persist this new date into `publishAt` via the schedule action. We should agree either to clear it or always keep it aligned. But of course the schedule action can also be triggered outwith studio, so guaranteeing they match isn't possible